styles/style.css

```diff
@@ -0,0 +1,166 @@
+/*Fonts*/
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+/**/
+/*nullstyle*/
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+*,
+*:before,
+*:after {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+:focus,
+:active {
+  outline: none;
+}
+
+a:focus,
+a:active {
+  outline: none;
+}
+
+nav,
+footer,
+header,
+aside {
+  display: block;
+}
+
+html,
+body {
+  height: 100%;
+  width: 100%;
+  font-size: 100%;
+  line-height: 1;
+  font-size: 14px;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+input,
+button,
+textarea {
+  font-family: inherit;
+}
+
+input::-ms-clear {
+  display: none;
+}
+
+button {
+  cursor: pointer;
+}
+
+button::-moz-focus-inner {
+  padding: 0;
+  border: 0;
+}
+
+a,
+a:visited {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: none;
+}
+
+ul li {
+  list-style: none;
+}
+
+img {
+  vertical-align: top;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-size: inherit;
+  font-weight: 400;
+}
+
+/*--------------------*/
+
+.subtitle {
+  font-family: Roboto;
+  font-size: 24px;
+  line-height: 28px;
+  color: #383838;
+}
+.wrapper {
+  display: flex;
+  flex-direction: column;
+}
+.title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 120px 0 0px 0;
+  font-family: Roboto;
+  font-weight: 500;
+  font-size: 40px;
+  line-height: 47px;
+  color: #383838;
+  text-align: center;
+}
+.title:after {
+  text-align: center;
+  content: "";
+  margin: 20px 0 0 0;
+  display: block;
+  width: 100px;
+  height: 4px;
+  background: #f5bd41;
+}
+.features {
+  display: flex;
+  justify-content: space-between;
+  margin: 80px 0 122px 0;
+  align-items: center;
+}
+.features__support {
+  text-align: center;
+  margin: 0 0 0 104px;
+}
+img.support__img {
+}
+.support__title {
+  margin: 48px 0 0 0;
+}
+.features__communication {
+  text-align: center;
+}
+img.communication__img {
+}
+.communication__title {
+  margin: 48px 0 0 0;
+}
+.features__book {
+  text-align: center;
+}
+img.book__img {
+}
+.book__title {
```

Comment:
You have some different names for classes, that contains the same properties, so it is better to create class .item_title and reuse it